### ISSUE TITLE
Error without a message on `init`

### ISSUE THREAD

**Issue by GalCohen** [arthurpalves/coherent-swift#34]
Hi there! Great tool! Really looking forward to incorporating it in my build process. Running into an issue on the init command: 

- coherent-swift 0.5.0
- installed through SwiftPM
```
❯ swift --version
Apple Swift version 5.2.4 (swiftlang-1103.0.32.9 clang-1103.0.32.53)
Target: x86_64-apple-darwin19.4.0
```

Here's the output

```
❯  swift run coherent-swift init
INFO  [2020-05-29 11:00:33]: ▸ --------------------------------------------------------------------------------------
INFO  [2020-05-29 11:00:33]: ▸ $ coherent-swift init
INFO  [2020-05-29 11:00:33]: ▸ --------------------------------------------------------------------------------------
INFO  [2020-05-29 11:00:33]: ▸ Generating specs file in: ./coherent-swift.yml
ERROR [2020-05-29 11:00:33]: ▸ --------------------------------------------------------------------------------------
ERROR [2020-05-29 11:00:33]: ▸ Error:
ERROR [2020-05-29 11:00:33]: ▸ --------------------------------------------------------------------------------------
Couldn't generate YAML specs
```

**Comment by GalCohen** [comment]
Interestingly enough, it works great when installed through `brew`

**Comment by arthurpalves** [comment]
Hi @GalCohen 
I think I know the issue, its about the templates’ path, its assuming it is on `/usr/local/lib` by default, which only happens through brew / make install.

I will provide a fix soon!

**Comment by arthurpalves** [comment]
This was fixed as part of [Release 0.5.1](https://github.com/arthurpalves/coherent-swift/releases/tag/0.5.1) 🎉
